src/IndexedDbProvider.ts

```diff
@@ -720,22 +720,24 @@ class IndexedDbIndex extends FullTextSearchHelpers.DbIndexFTSFromRangeQueries {
         }
     }
 
-    getAll(reverse?: boolean, limit?: number, offset?: number): SyncTasks.Promise<ItemType[]> {
+    getAll(reverseOrSortOrder?: boolean | NoSqlProvider.QuerySortOrder, limit?: number, offset?: number): SyncTasks.Promise<ItemType[]> {
         // ************************* Don't change this null to undefined, IE chokes on it... *****************************
         // ************************* Don't change this null to undefined, IE chokes on it... *****************************
         // ************************* Don't change this null to undefined, IE chokes on it... *****************************
+        const reverse = reverseOrSortOrder === false || reverseOrSortOrder === NoSqlProvider.QuerySortOrder.Reverse;
```

Comment:
Nit: Move `const` above comment. The farther the comment is from the `null!!!` then the less likely the reader will notice and read the comment.